core/admin/mailu/schemas.py

```diff
@@ -1176,7 +1176,7 @@ class Meta:
         model = models.User
         load_instance = True
         include_relationships = True
-        exclude = ['_email', 'domain', 'localpart', 'domain_name', 'quota_bytes_used']
```

Comment:
This breaks the command line [mailu config-export](https://mailu.io/2.0/cli.html#config-export)
This cli command now outputs this value (quota_bytes_used) and that is not allowed. 
The output from mailu config-export must be a valid yml file that can be imported again.

Reply:
The exclude line must not be removed.
The config export is designed to only export configuration data, not runtime data.

Reply:
Why was this change added at all?
If I only update `user_fields_get = api.model('UserGet',` with the quota_bytes_used it already works for me.